app/views/admin/metrics/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+  <div class="col-xs-12">
+    <h1>Metrics</h1>
+  </div>
+</div>
+
+<table class="table table-bordered table-striped">
+  <thead>
+    <tr>
+      <th>Date</th>
+      <th></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @snapshots.each do |snapshot| %>
+      <tr class="t-snapshot-table-row">
+        <td style="vertical-align: middle"><%= snapshot.created_at.strftime('%e %B %Y') %></td>
```

Comment:
inline style... maybe want to do something like:

`<td class="table__cell--middle">...</td>`

Or some other BEM name.